projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/create.py

```diff
@@ -30,7 +30,13 @@ def __init__(self, rest_api_url: str):
 
     @ApiClientErrorDecorator()
     def create_job(
-        self, name: str, team: str, path: str, cloud: bool, local: bool
+        self,
+        name: str,
+        team: str,
+        path: str,
+        cloud: bool,
+        local: bool,
+        sample_job_directory=None,
```

Comment:
The default value is not None, but the actual default sample_job dir in vdk-control-cli. 
The advice is to keep a clean user-facing api surface, that reflects the actual defaults.

Reply:
resolving this since it was merged in a separate PR